scss/_variables.scss

```diff
@@ -438,7 +438,7 @@ $enable-validation-icons:     true !default;
 $enable-negative-margins:     false !default;
 $enable-deprecation-messages: false !default;
 $enable-important-utilities:  true !default;
-$enable-fixed-header:         true !default; // Boosted mod: used to apply scroll-padding-top
+$enable-fixed-header:         false !default; // Boosted mod: used to apply scroll-padding-top
```

Comment:
Are you sure that the default value of `$enable-fixed-header` should be `false`?

Taking https://deploy-preview-1274--boosted.netlify.app/docs/5.3/examples/navbar-sticky/ as an example which doesn't work as expected. Maybe https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/site/content/docs/5.3/examples/navbar-sticky/navbar-sticky.css needs to be modified in this PR as well.